.github/workflows/release.yml

```diff
@@ -68,3 +68,30 @@ jobs:
       - env:
           GH_TOKEN: ${{ github.token }}
         run: gh release upload ${{ github.ref_name }} ${{ steps.archive.outputs.path }}
+  binaries:
+    needs: release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - build: linux-64-bit
+            os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+            prebuild: "true"
+            command: cargo
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@nightly
```

Comment:
Haha, the library builds fine on stable actually! We only use the nightly toolchain just because the cross compiling helper crate needs it, but we don't add the nightly flag when building the library itself